constants/post.ts

```diff
@@ -0,0 +1,2 @@
+export const NUMBER_OF_POSTS = 6;
+export const POPULAR_POSTS_TERM_FROM_IN_MONTH = 12;
```

Comment:
It doesn't make sense 🤔 What is this for?

Reply:
I thought the popular list should have term, like "popular in 12 months" and we can change the term when users post more frequently.